kvbc/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ add_library(kvbc  src/ClientImp.cpp
                   src/sparse_merkle/walker.cpp
 )
 
-target_link_libraries(kvbc PUBLIC threshsign util concordbft_storage)
+target_link_libraries(kvbc PUBLIC threshsign util concordbft_storage corebft)
```

Comment:
consider arget_link_libraries(kvbc PUBLIC corebft)

since corebft is linked with threshsign util concordbft_storage

Reply:
I am thinking that if the `corebft` depdendencies change in the future, `kvbc` will still work if we leave them explicit here.

Reply:
it's up to you